worlds/ff6wc/__init__.py

```diff
@@ -369,6 +376,67 @@ def create_items(self) -> None:
             self.create_event("Victory"))
         self.multiworld.completion_condition[self.player] = lambda state: state.has("Victory", self.player)
 
+        # Check to see what the Item Rewards are to populate the "dead" checks
+        # NOTE: most of this code is located in WorldsCollide/args/items.py during the process function
+        self.item_rewards = []
+        self.item_nonrewards = []
+        # if -ir in flagstring, then user specified item rewards
+        if "-ir" in self.options.Flagstring.value.split(" "):
+            # get the item reward string between the -ir flag & the next one
+            item_reward = " ".join(self.options.Flagstring.value.split("-ir")[1:]).split("-")[0].strip()
```

Comment:
I'm trying to understand this flagstring parsing here...

The `join` call only does something if `self.options.Flagstring.value.split("-ir")[1:]` has more than 1 value, right?
which only happens if `self.options.Flagstring.value.split("-ir")` has more than 2 values
which only happens if `"-ir"` appears more than once.

Are we supporting more than 1 `-ir` in a flagstring?
It looks like it wouldn't work, because it's splitting by `","` after joining by `" "`.

Reply:
Can any flag appear more than once in the flagstring?

Reply:
I don't believe that we allow a flagstring to have more than 1 value, but perhaps it's the case that whomever wrote the parser for the flagstring (original code dev) may have chosen to accept the first valid flagstring value & ignore the rest. In general, the same flagstring specified multiple times is not a good thing.